bindgen/lib.rs

```diff
@@ -348,6 +348,13 @@ impl Builder {
         }
 
         // Transform input headers to arguments on the clang command line.
+        self.options.fallback_clang_args = self
+            .options
+            .clang_args
+            .iter()
+            .filter(|arg| !arg.contains("-MMD") && !arg.contains("-MD"))
```

Comment:
Should this be `starts_with`? Also shouldn't we also skip the long-version of these? (`--write-user-dependencies` / `--user-dependencies`)?